Samples/PowerShell/ADLSUser/Set-AdlsAccess.ps1

```diff
@@ -134,49 +185,81 @@ function copyacls
 # This script assumes the following:
 # 1. The Azure PowerShell environment is installed
 # 2. The current session has already run "Login-AzureRMAccount" with a user account that has permissions to the specified ADLS account
-try
-{   if($EntityType -ieq "other" -and !([string]::IsNullOrEmpty($EntityIdToAdd)))
-	{
-		throw "EntityIdToAdd is not supported when modifying permissions for 'Other' entity types"
-	}
-	
-	if($EntityType -ine "other" -and [string]::IsNullOrEmpty($EntityIdToAdd))
-	{
-		throw "EntityIdToAdd is required when modifying permissions for 'user' and 'group' entity types"
-	}
+
+switch ($PsCmdlet.ParameterSetName) 
+{ 
+    “SetOwner”  {
+        try
+        {
+            if($EntityType -ieq "other")
+	        {
+		        throw "SetOwner is not supported when 'Other' EntityType is specified"
+	        }
 	
-	$ignored = [Guid]::Empty
-	if($EntityType -ine "other" -and !([Guid]::TryParse($EntityIdToAdd, [ref]$ignored)))
-	{
-		throw "EntityIdToAdd must be a valid GUID. EntityIdToAdd value was: $EntityIdToAdd"
-	}
+	        if($EntityType -ine "other" -and [string]::IsNullOrEmpty($EntityId))
```

Comment:
you don't need to check for entityType here because you already checked for "other" above.